court_ball_game/tests.py

```diff
@@ -0,0 +1,61 @@
+from django.contrib.auth import get_user_model
+from .models import CourtBallGame
+from court.models import Court
+from player.models import BallGame
+from decimal import Decimal
+from django.db.utils import IntegrityError
+from django.core.exceptions import ValidationError
+import pytest
+
+
+@pytest.mark.django_db
+class TestCourtBallGameModel:
```

Comment:
instead of declaring test_court in each function try using pytest.fixture

Reply:
Thanks for the tip, will be fixed.

Reply:
Done

Reply:
I don't see the unique_together stuff, not the test for it.